tests/driver_motor_driver/Makefile

```diff
@@ -0,0 +1,16 @@
+include ../Makefile.tests_common
```

Comment:
Place the include after setting the board, otherwise the default setting below gets ignored.